client/src/components/glossary_components.tsx

```diff
@@ -11,27 +11,52 @@ import { is_a11y_mode } from "src/core/injected_build_constants";
 import { IconQuestion } from "src/icons/icons";
 import { glossary_href } from "src/link_utils";
 
-const GlossaryTooltipWrapper = ({ id, children, no_bottom_border }) =>
+interface GlossaryTooltipWrapperProps {
+  id: string;
+  children?: React.ReactNode;
+  no_bottom_border?: boolean;
+}
+
+interface GlossaryIconProps {
+  id: string;
+  alternate_text?: string;
+  arrow_selector?: any;
+  inner_selector?: any; // i don't know how to do the types for the selectors
```

Comment:
Look like they're totally unused, probably legacy props someone missed cleaning up while refactoring this component? Not used in `GlossaryIcon`, and no consumers of `GlossaryIcon` are trying to pass them in either. Safe to drop them :+1: